hydroshare/urls.py

```diff
@@ -143,7 +144,7 @@
     # one out.
 
     # url("^$", direct_to_template, {"template": "index.html"}, name="home"),
-    url(r"^tests/$", direct_to_template, {"template": "tests.html"}, name="tests"),
+    # QUNIT_TESTS_OFF
```

Comment:
This is a modified urls.py that should not be committed. The QUNIT_TESTS_OFF line is generated by ./hsctl during a build.

Reply:
Revert to the develop version.